portal-impl/src/portal-osgi-configuration.properties

```diff
@@ -2029,8 +2029,6 @@
     #
     # Env: LIFERAY_CONFIGURATION_PERIOD_OVERRIDE_PERIOD_COM_PERIOD_LIFERAY_PERIOD_DYNAMIC_PERIOD_DATA_PERIOD_MAPPING_PERIOD_FORM_PERIOD_WEB_PERIOD_INTERNAL_PERIOD_CONFIGURATION_PERIOD__UPPERCASEF__UPPERCASEF__UPPERCASES_UBMISSIONS_UPPERCASES_ETTINGS_UPPERCASEC_ONFIGURATION_UNDERLINE_EXPIRATION_UPPERCASED_ATE_UPPERCASEE_NABLED
     #
-    #configuration.override.com.liferay.dynamic.data.mapping.form.web.internal.configuration.FFSubmissionsSettingsConfiguration_expirationDateEnabled=
```

Comment:
You don't have to remove this manually.

Reply:
Good to know, so I will remove this change, thanks!